packages/sidenav/src/ux-sidenav-drawer/ux-sidenav-drawer.scss

```diff
@@ -0,0 +1,45 @@
+@import '../common.scss';
+
+ux-sidenav-drawer {
+  position: absolute;
+  z-index: 3;
+  background-color: var(--aurelia-ux--design-control-background, white);
+  color: var(--aurelia-ux--design-control-foreground, black);
```

Comment:
I think it would be better to have a themable properties for background/foregound of the drawer and if nothing provided defaults to `surface` pair.

```css
background-color: var(--aurelia-ux--drawer-background, var(--aurelia-ux--design-surface-background, #F5F5F5));
color: var(--aurelia-ux--drawer-foreground, var(--aurelia-ux--design-surface-foreground, #212121));
```

Reply:
Done

Reply:
Cool thanks. As defaults I would use the `surface` pair instead of the `control` pair. But you might have a reason why you defaults to control ?

Reply:
Coz I've found them first :)

Reply:
Actually, no... With surface colours the drawers will blend with surroundings

Reply:
In the theming page the components are demoed inside a card -> which is a surface. So yes, in this scenario it will blend with surroundings and that's where you need a `theme.bind="{}"` on your sidenav to change this style.

To be honest I don't think that this demo inside a card is a great design. But it's what we have yet (and it's my fault btw). Hopefully we'll have a better demo page soon ;-)

But by design a sidenav is more surface than a control. A sidenav should not be used inside card anyway ?!?

Reply:
Changed